examples/decorators/uniffi.toml

```diff
@@ -0,0 +1,43 @@
+[bindings.python]
+cdylib_name = "wrapper_types"
+
+[bindings.ruby]
+cdylib_name = "wrapper_types"
+
+[bindings.swift]
+cdylib_name = "wrapper_types"
+
+[bindings.kotlin]
+cdylib_name = "wrapper_types"
+
+# Defining the log-errors decorator for the Python bindings
+[decorators.log-errors.python]
+imports = ["logging"]
+code = """
+error_logger = logging.getLogger("errors")
+
+def logErrors(func):
```

Comment:
So stretching the other comments further, this should really be `def logErrors(func, *args, **kw): ...` - but how do we spell that in a generic way?

Reply:
I was thinking that we would always input a function with no args and UniFFI would create a closure for that.

Reply:
Right, for some reason I misunderstood that, including in James's patch. I think some of the confusion came from the fact they were specified in the UDL, and I guess because (IIUC), decorators in Python will want the `*args, **kwargs` in most cases. But here, we will *never* have args specified, and it will forever be impossible for a "decorator" to get access to the args? Also still not clear to me quite yet how return types should be managed efficiently, but as you say, maybe that all "just works"